app/Models/Action.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Rogue\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class Action extends Model
+{
+    use SoftDeletes;
+
+    /**
+     * The attributes that should be mutated to dates.
+     *
+     * @var array
+     */
+    protected $dates = ['deleted_at', 'start_date', 'end_date'];
```

Comment:
Is `deleted_at` something that's actually visible on the model, or just used behind the scenes for soft deletes?

Reply:
ah great call @DFurnes ! Deleted this whole thing bc there is no `start_date` or `end_date` for this - was copied over by mistake :)

Reply:
Ahhh I didn't even notice that part!! 🙃

Reply:
@chloealee I think this snuck back in when getting rid of that other sneaky file!